code/modules/events/disease_outbreak.dm

```diff
@@ -83,81 +117,251 @@
 	name = "Disease Outbreak: Advanced"
 	typepath = /datum/round_event/disease_outbreak/advanced
 	category = EVENT_CATEGORY_HEALTH
-	description = "An 'advanced' disease will infect some members of the crew." //These are the ones that get viro lynched!
+	weight = 10
+	description = "An 'advanced' disease will infect some members of the crew."
 	admin_setup = /datum/event_admin_setup/disease_outbreak/advanced
 
+/datum/event_admin_setup/disease_outbreak/advanced
+	///Admin selected custom severity rating for the event
+	var/chosen_severity
+	///Admin selected custom value for the maximum symptoms this virus should have
+	var/chosen_max_symptoms
+
+/datum/event_admin_setup/disease_outbreak/advanced/prompt_admins()
+	var/candidate_count = candidate_check()
+	if(!candidate_check())
+		tgui_alert(usr, "There are no candidates eligible to recieve a disease!", "Error")
+		return ADMIN_CANCEL_EVENT
+	tgui_alert(usr, "[candidate_count] candidates found!", "Disease Outbreak")
+
+	/**
+	 * Admin virus customization
+	 *
+	 * If the admin wishes, give them the opportunity to select the severity and number of symptoms.
+	 */
+	if(tgui_alert(usr,"Customize your virus?", "Glorified Debug Tool", list("Yes", "No")) == "Yes")
```

Comment:
Add this multi-line documentation to be above the proc itself, not down here because it just adds to clutter in the middle of working code (though you probably could have a single line `//` comment if you wanted in its place)